traffic_monitor/datareq/crstate.go

```diff
@@ -21,6 +21,7 @@ package datareq
 
 import (
 	"fmt"
+	"github.com/apache/trafficcontrol/traffic_monitor/health"
```

Comment:
Imports starting with `github.com/apache/trafficcontrol` should be grouped separately

Reply:
found an goland option to enable this automatically, fixed.